pkg/cmd/server/origin/controller/network_sdn.go

```diff
@@ -35,6 +35,12 @@ func (c *SDNControllerConfig) RunController(ctx ControllerContext) (bool, error)
 	}
 
 	networkInformers.Start(utilwait.NeverStop)
+	syncStatus := networkInformers.WaitForCacheSync(utilwait.NeverStop)
+	for informerType, ok := range syncStatus {
```

Comment:
I was assuming based on @smarterclayton 's "like other controllers" comment that this would be doing the same thing as some other code somewhere else, but I don't see any other code doing anything like this. Where did this come from?

Reply:
This code actually waits for all network informer caches to be populated. This is helpful when we know we are going to use most of the network informers (hostsubnet, netnamespace, etc.). Kube informers supports many resources and instead of waiting for all the informers, we can wait just for the ones that we care about using kcache.WaitForCacheSync(), that's the pattern used by other controllers.